src/nl/Stratego/Speelstuk.java

```diff
@@ -56,4 +56,12 @@ public String getNaam() {
     public void setNaam(String naam) {
         this.naam = naam;
     }
+
+    public int getTeam() {
+        return team;
+    }
+
+    public void setTeam(int team) {
```

Comment:
Is een setTeam verstandig? Je zou dan het team kunnen aanpassen tijdens het spel. Of zou dit nodig zijn later voor de database dat je hem opslaat en opnieuw laad?

Reply:
Ik heb geen idee, had een getter en setter gemaakt, maar een getter is inderdaad voldoende en alleen nodig. een setter hebben we niet nodig (tenzij we dat later nodig hebben idd). Maar dan kunnen we dat nog snel genoeg aanmaken!